FEATURES.md

```diff
@@ -0,0 +1,175 @@
+# Features
+
+This document describes a list of use-cases and features that this library provides.
+
+## Use Cases
```

Comment:
The order of use-cases (from UC 1 to UC 4) also defines the prioritization from my point of view.